main.js

```diff
@@ -0,0 +1,87 @@
+const Board = function (x) {
+x = [["1", "2", "3"],
+["4", "5", "6"],
+["7", "8", "9"]];
+return x;
+}
+
+var gameBoard;
+var playerTurn;
+var moveCount = 0;
+
+function printBoard() {
+  var output = gameBoard[0].join("") + "\n" + gameBoard[1].join("") + "\n" + gameBoard[2].join("");
+  console.log(output);
+}
+
+function newGame() {
+  moveCount = 0;
+  playerTurn = "O"
+  gameBoard = new Board;
+  console.log("\n\n\n\n\n" + "NEW GAME");
+  printBoard();
+  console.log("X goes first. Use 'play()' to take a turn.")
+}
+
+function checkWinner() {
+  if (gameBoard[0][0] == gameBoard[0][1] && gameBoard[0][0] == gameBoard[0][2]) {
```

Comment:
This works, but it's hardcoded. A cleaner and more concise way would be to iterate through our 2D array instead of hardcoding indices.